### PR TITLE
Update xamarin.md

### PR DESCRIPTION
**Pull request by JeremyLikness**
Documentation related to [dotnet/efcore #22559](https://github.com/dotnet/efcore/issues/22559). May address [dotnet/efore #10963](https://github.com/dotnet/efcore/issues/10963).

### CODE REVIEW

File: entity-framework/core/get-started/xamarin.md [hunk 1]
```diff
@@ -187,6 +187,23 @@ using (var blogContext = new BloggingContext())
...
+Entity Framework Core sometimes generates dynamic code for performance reasons. This is not supported on the iOS platform and may cause the application to crash. The Xamarin team built an interpretor to handle these scenarios. It should be configured in your `.csproj` file in a `PropertyGroup`. Use a `Condition` to ensure the group is only valid for iPhone release builds:
+
+```xml
+<PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|iPhone' ">
+</PropertyGroup>
+```
+
+Then add the `--interpreter` flag as an extra argument inside the group:
+
+```xml
```
Comment (smitpatel) [inline entity-framework/core/get-started/xamarin.md:201]: Combining both xml blocks would make it clearer what is happening overall. Also easy to copy paste.

File: entity-framework/core/get-started/xamarin.md [hunk 2]
```diff
@@ -187,6 +187,23 @@ using (var blogContext = new BloggingContext())
...
+<CodesignExtraArgs>--interpreter</CodesignExtraArgs>
+```
+
+More information is available in the [interpreter repository](https://github.com/spouliot/interpreter).
```
Comment (smitpatel) [inline entity-framework/core/get-started/xamarin.md:205]: Is it microsoft product? If not we should call out accordingly.

File: entity-framework/core/get-started/xamarin.md [hunk 3]
```diff
@@ -187,6 +187,23 @@ using (var blogContext = new BloggingContext())
 }
 ```
 
+## iPhone Configuration
+
+Entity Framework Core sometimes generates dynamic code for performance reasons. This is not supported on the iOS platform and may cause the application to crash. The Xamarin team built an interpretor to handle these scenarios. It should be configured in your `.csproj` file in a `PropertyGroup`. Use a `Condition` to ensure the group is only valid for iPhone release builds:
```
Comment (lion92pl) [inline entity-framework/core/get-started/xamarin.md:192]: interpretor -> interpreter

File: entity-framework/core/get-started/xamarin.md [hunk 4]
```diff
@@ -187,6 +187,23 @@ using (var blogContext = new BloggingContext())
...
+
+```xml
+<PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|iPhone' ">
+</PropertyGroup>
+```
+
+Then add the `--interpreter` flag as an extra argument inside the group:
+
+```xml
+<CodesignExtraArgs>--interpreter</CodesignExtraArgs>
```
Comment (lion92pl) [inline entity-framework/core/get-started/xamarin.md:202]: Shouldn't it be `<MtouchExtraArgs>--interpreter</MtouchExtraArgs>`?
See my comment in #2666

File: entity-framework/core/get-started/xamarin.md (hunk 2 above)
Comment (JeremyLikness) [reply entity-framework/core/get-started/xamarin.md:205]: Good question. It was built by someone on the Xamarin team but it is in their personal repo. Do we have a standard boilerplate?

File: entity-framework/core/get-started/xamarin.md (hunk 4 above)
Comment (JeremyLikness) [reply entity-framework/core/get-started/xamarin.md:202]: Good question. I'm confirming with @codemillmatt who has been working on the sample.

Comment (codemillmatt) [reply entity-framework/core/get-started/xamarin.md:202]: Turns out the interpreter isn't the issue here after all - it is a linking issue. I'll update the code to make sure EF isn't linked out.

**Comment by codemillmatt** [comment]
Please see PR #2702 that one updates the documentation for the quick start. If #2702 checks out - then we can close this one without merging it.

File: entity-framework/core/get-started/xamarin.md (hunk 3 above)
Comment (Rick-Anderson) [inline entity-framework/core/get-started/xamarin.md:192]:
```suggestion
Entity Framework Core sometimes generates dynamic code for performance reasons. This isn't supported on the iOS platform and may cause the app to crash. The Xamarin team built an interpreter to handle these scenarios. It should be configured in the `.csproj` file in a `PropertyGroup`. Use a `Condition` to ensure the group is only valid for iPhone release builds:
```

Comment (Rick-Anderson) [reply entity-framework/core/get-started/xamarin.md:192]: Per MS style guide, prefer contractions. Also the style guide insists files be `*italic*`, but for project files, I prefer code fencing as you've done.

**Comment by JeremyLikness** [comment]
Superceded by #2702

**Pull request closed without merging**